qlib/rl/data/native.py

```diff
@@ -82,18 +81,19 @@ def load_backtest_data(
     trade_exchange: Exchange,
     trade_range: TradeRange,
 ) -> IntradayBacktestData:
-    data = cast(
-        IndexData,
-        trade_exchange.get_deal_price(
-            stock_id=order.stock_id,
-            start_time=order.date,
-            end_time=order.date + ONE_DAY - EPS_T,
-            direction=order.direction,
-            method=None,
-        ),
+    tmp_data = D.features(
```

Comment:
I think making exchange return data without missing will make it more elegant.
Please add TODO here.